<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tableaquery</class>
 <widget class="QDialog" name="tableaquery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>关键字</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="upButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>升序</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="downButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>降序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>排序方式</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ridButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>房间号</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="cidButton_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>顾客id</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="cnameButton_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>顾客名</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="startTButton_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>开始时间</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="endTButton_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>结束时间</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="nowTButton_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>现在时间</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>296</width>
     <height>236</height>
    </rect>
   </property>
   <property name="firstDayOfWeek">
    <enum>Qt::Sunday</enum>
   </property>
   <property name="horizontalHeaderFormat">
    <enum>QCalendarWidget::ShortDayNames</enum>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
   <property name="navigationBarVisible">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
